.eslintrc-base.json

```diff
@@ -61,7 +61,6 @@
     "jsx-a11y/no-noninteractive-element-interactions": "error",
     "jsx-a11y/no-noninteractive-element-to-interactive-role": "error",
     "jsx-a11y/no-noninteractive-tabindex": "error",
-    "jsx-a11y/no-onchange": "error",
```

Comment:
This rule gave me an error because we are using `onChange` in the stream dropdown menu. I don't see anything wrong with that, and decide to remove the rule as it is deprecated and no longer recommended. See https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md